core/domain/suggestion_registry.py

```diff
@@ -329,7 +329,8 @@ def populate_old_value_of_change(self):
         if self.change.state_name not in exploration.states:
             # As the state doesn't exist now, we cannot find the content of the
             # state to populate the old_value field. So we set it as None.
-            old_content = None
```

Comment:
Oops, my bad! I read my comment above this and realised it has to be None. Sorry about the confusion. This exception would break some of the flows (like viewing suggestions to old states), and hence we had kept it as None. Can you please revert this change. Sorry for the confusion.

Reply:
Done